manifests/prometheus.yml

```diff
@@ -0,0 +1,547 @@
+---
+meta:
+  passthru: ((grab params.bosh_exodus_path || params.bosh || params.env))
+  default:
+    azs: [z1]
+  prometheus_port:   (( grab params.prometheus_port   || 8080 ))
+  grafana_port:      (( grab param.grafana_port       || 443 ))
+  alertmanager_port: (( grab params.alertmanager_port || 8082 ))
+
+exodus:
+  prometheus_url:   (( concat params.external_domain ":" meta.prometheus_port    ))
+  grafana_url:      (( concat params.external_domain ":" meta.grafana_port       ))
+  alertmanager_url: (( concat params.external_domain ":" meta.alertmanager_port  ))
+
```

Comment:
You need to define `(( param ... ))` operations for parameters that lack defaults, like `params.static_ip`